ddtrace/_trace/tracer.py

```diff
@@ -1150,11 +1150,7 @@ def _use_sync_mode() -> bool:
         - Google Cloud Functions and Azure Consumption Plan Functions have a mini-agent spun up by the tracer.
```

Comment:
nit: might as well change this from Azure Consumption Plan Functions -> Azure Functions